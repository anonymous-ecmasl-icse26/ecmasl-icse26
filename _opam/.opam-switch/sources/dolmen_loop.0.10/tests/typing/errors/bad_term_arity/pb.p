
tff(test1, type, (p : ($i * $i) > $o)).

tff(test2, axiom, p(a) ).

