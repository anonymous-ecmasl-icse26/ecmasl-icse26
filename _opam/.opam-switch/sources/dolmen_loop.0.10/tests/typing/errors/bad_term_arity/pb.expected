File "tests/typing/errors/bad_term_arity/pb.p", line 4, character 18-22:
4 | tff(test2, axiom, p(a) ).
                      ^^^^
Error Bad arity: expected 2 but got 1 arguments for function
      p : $i -> $i -> Prop
